bridge-adaptivity/bridge_lti/consumer.py

```diff
@@ -25,30 +26,41 @@ def tool_config(request):
     return HttpResponse(lti_tool_config.to_xml(), content_type='text/xml')
 
 
-def content_source(request, pk):
+def source_preview(request):
     """
-    Simple view to render Source's component shared through LTI.
+    Simple view to render content Source's block shared through LTI.
     """
-    lti_source = LtiSource.objects.get(id=pk)
+    activity_id = request.GET.get('activity_id')
+    if activity_id:
+        activity = Activity.objects.get(id=activity_id)
+        source_name = activity.source_name
+        source_lti_url = activity.source_launch_url
+    else:
+        source_name = request.GET.get('source_name')
+        source_lti_url = request.GET.get('source_lti_url').replace(u' ', u'+')  # Django strips plus sign
```

Comment:
typo s/Django strips plus sign/Django strips spaces/

Reply:
Actually, Django converts plus signs to spaces while URL query params parsing.

Reply:
Yep, and nevertheless s/strips plus sign/converts plus sign to space/